.circleci/config.yml

```diff
@@ -154,7 +154,29 @@ workflows:
           working_directory: sites/partners
           yarn: true
           build: |
+            echo 'export FEATURE_LISTINGS_APPROVAL=FALSE' >> "$BASH_ENV"
+            source "$BASH_ENV"
             yarn test:backend:core:dbsetup
-          start: yarn dev:all-cypress
+          start: |
```

Comment:
Do you need to add this configuration to the .github/workflows so that the new test runs?

Reply:
Yeah I made a separate ticket in the epic for that, just because it may involve tinkering / more commits